tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -12,8 +12,8 @@ import {
   logger,
   writeJson,
   updateProjectConfiguration,
+  serializeJson,
```

Comment:
nrwl team is moving utils gradually to nx devkit 🙌

- more reasoning/info https://github.com/nrwl/nx/issues/6153